_posts/2023-04-19-Announcing-Data-Prepper-2.2.0.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.2.0"
+authors:
+- dlv
+date: 2023-04-19 14:30:00 -0500
+categories:
+  - releases
+meta_keywords: Data Prepper 2.1.0
+meta_description: Data Prepper 2.1.0 improves data delivery assurances with end-to-end acknowledgements and an S3 DLQ.
```

Comment:
```suggestion
meta_description: Data Prepper 2.2.0 improves data delivery assurances with end-to-end acknowledgements and an S3 DLQ.
```

Reply:
**meta_description:** Read the blog to learn how Data Prepper 2.1.0 improves reliability and writes documents from failed events directly in to an Amazon S3 dead letter queue (DLQ).